doc/examples/applications/plot_rank_filters.py

```diff
@@ -401,6 +403,49 @@
 
 plt.tight_layout()
 
+######################################################################
+# The example compares the local threshold with the global threshold in 3D
+
+def normalize_pixels(i):
+    i = 256*(i/np.amax(i))
+    return i.astype(np.uint8)
+
+p8 = data.brain()
+
+p8 = normalize_pixels(p8)
+
+radius = 50
+selem = ball(radius)
+
+# t_loc_otsu is an image
+t_loc_otsu = otsu(p8, selem)
```

Comment:
I changed `otsu` to `rank.otsu` since rank was at one point imported by the contrast enhancement demo code. 
I cannot really remove the rank filters import because it is also used by the old 2d otsu demo. In fact almost all demo (except contrast enhancement) uses something like `from skimage.filters.rank import xxx`

Reply:
I changed `otsu` to `rank.otsu` because it is at one point imported by the contrast enhancement demo code. 
I should not really remove the rank filter import because the old 2d otsu demo also uses it. In fact, almost all filter demos use the format `from skimage.filters.rank import xxx`